spec/dummy/app/assets/config/manifest.js

```diff
@@ -2,6 +2,7 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
 //= link_directory ../stylesheets .css
+//= link application.css
```

Comment:
Is this intentional? / Where does this come from?

Reply:
The unit tests don't run locally for me unless I put this in. 
It's under spec folder for a dummy app so should not be impacting any production code